Gruntfile.js

```diff
@@ -407,7 +407,7 @@ module.exports = function(grunt) {
     shell: {
         'build-jsx': {
             command: [
-              './node_modules/react-tools/bin/jsx -x jsx app/addons/ app/addons/',
+              'node ./node_modules/react-tools/bin/jsx -x jsx app/addons/ app/addons/',
                 'rm -rf <%= src_path %>/js/app/views/.module-cache/'
```

Comment:
Actually that whole line can be removed since it actually doesn't work. The `.module-cache` isn't saved to that directory.